src/buza/static/buza/css/questions.css

```diff
@@ -25,22 +25,14 @@
     color: DarkOrchid;
 }
 
-.question__ask_question {
+.question__edit_question, .question__ask_question, input[type=button], input[type=submit], input[type=reset] {
```

Comment:
We should avoid targeting `input` or other elements directly: our styles should rather target a reusable class.

Reply:
this would mean that we have to for format the form field invidually. Is that okay?

Reply:
Well, crispy-forms will replace the need for manual formatting: it's the best of both worlds. :)

I'll tackle that here and show!

Reply:
awesome!